docs/content/theming.md

```diff
@@ -176,6 +176,16 @@ function Example() {
 }
 ```
 
+#### `preventSSRMismatch` prop
+
+For usage with server side rendering, it is recommended to pass `preventSSRMismatch` to ensure the rendered output from the server and browser match even when they resolve "auto" color mode differently.
```

Comment:
We generally try to avoid passive voice (i.e. "it is recommended")
```suggestion
If you are doing server-side rendering, pass the `preventSSRMismatch` prop to ensure the rendered output from the server and browser match even when they resolve "auto" color mode differently.
```